circuit_tutorials/gnark/README.md

```diff
@@ -0,0 +1,70 @@
+# Sample Gnark Circuits
```

Comment:
I think all of the `circuit_tutorials/{framework}/README.md` contain information that would be better suited in `circuit_database` or the docs.  It might confuse people browsing this directory. My suggestion is to totally remove any section that indicates general Sindri requirements, such as `##circuit requirements`, `##supporting files`, etc